core/domain/prod_validation_jobs_one_off_test.py

```diff
@@ -6209,6 +6211,2387 @@ def test_model_with_invalid_story_related_property(self):
+        self.model_instance_0.commit(self.owner_id, '', [])
+        expected_output = [
+            (
+                u'[u\'failed validation check for uncategorized skill '
+                'ids check of TopicModel\', '
+                '[u\'Entity id 0: uncategorized skill id 0 is present '
+                'in subtopic for entity with id 1\']]'
+            ), u'[u\'fully-validated TopicModel\', 2]']
+        run_job_and_check_output(self, expected_output, sort=True)
+
+
+class TopicSnapshotMetadataModelValidatorTests(
+        test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(TopicSnapshotMetadataModelValidatorTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+
+        self.signup(USER_EMAIL, USER_NAME)
+        self.user_id = self.get_user_id_from_email(USER_EMAIL)
+
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.set_admins([self.ADMIN_USERNAME])
+
+        topics = [topic_domain.Topic.create_default_topic(
+            topic_id='%s' % i, name='topic%s' % i) for i in xrange(3)]
+
+        skills = [skill_domain.Skill.create_default_skill(
+            skill_id='%s' % i, description='skill%s' % i) for i in xrange(9)]
+
+        for skill in skills:
+            skill_services.save_new_skill(self.owner_id, skill)
+            skill_services.publish_skill(skill.id, self.admin_id)
+
+        stories = [story_domain.Story.create_default_story(
+            '%s' % i,
+            title='title %d',
+            corresponding_topic_id='%s' % (i / 2)
+        ) for i in xrange(6)]
+
+        for story in stories:
+            story_services.save_new_story(self.owner_id, story)
+
+        language_codes = ['ar', 'en', 'en']
+        for index, topic in enumerate(topics):
+            topic.language_code = language_codes[index]
+            topic.update_additional_story_ids(['%s' % (index * 2)])
+            topic.add_canonical_story('%s' % (index * 2 + 1))
+            topic.add_uncategorized_skill_id('%s' % (index * 3))
+            topic.add_uncategorized_skill_id('%s' % (index * 3 + 1))
+            topic.add_uncategorized_skill_id('%s' % (index * 3 + 2))
+            if index == 0:
+                topic_services.save_new_topic(self.user_id, topic)
+            else:
+                topic_services.save_new_topic(self.owner_id, topic)
+
+        self.model_instance_0 = (
+            topic_models.TopicSnapshotMetadataModel.get_by_id(
+                '0-1'))
+        self.model_instance_1 = (
+            topic_models.TopicSnapshotMetadataModel.get_by_id(
+                '1-1'))
+        self.model_instance_2 = (
+            topic_models.TopicSnapshotMetadataModel.get_by_id(
+                '2-1'))
+
+        self.job_class = prod_validation_jobs_one_off.TopicSnapshotMetadataModelAuditOneOffJob # pylint: disable=line-too-long
```

Comment:
Try to get rid of pragmas by breaking on the period, here and elsewhere.

Reply:
Done.